src/components/DatasetItem.vue

```diff
@@ -262,16 +262,15 @@
 
    apollo: {
      opticalImageUrl: {
-       query: opticalImageQuery,
+       query: thumbnailOptImageQuery,
        variables() {
          return {
-           datasetId: this.dataset.id,
-           zoom: 1.
+           datasetId: this.dataset.id
          };
        },
        fetchPolicy: 'network-only',
        result(res) {
-         this.opticalImageSmall = res.data.opticalImageUrl
+         this.opticalImageSmall = res.data.thumbnail[0]
```

Comment:
`thumbnail[0]` should be `thumbnailImage` here to match the field name defined in graphql.